files/galaxy/tpv/destinations.yml.j2

```diff
@@ -101,9 +100,7 @@ destinations:
       dependency_resolution: "none"
       submit_request_cpus: "{cores}"
       submit_request_memory: "{mem}"
-      # $defaults expands to "$galaxy_root:ro,$tool_directory:ro,$job_directory:ro,$working_directory:rw,$default_file_path:rw"
-      docker_volumes: "$defaults,/cvmfs/data.galaxyproject.org:ro"
-      singularity_volumes: "$defaults,/cvmfs/data.galaxyproject.org:ro"
+      docker_volumes: "$defaults,$job_directory:rw,$tool_directory:ro,{{ cvmfs.data.path }}:{{ cvmfs.data.docker_perm }}"
```

Comment:
Same question as before, why do we need `$job_directory` to be writable now? Also `$tool_directory:ro` should be included in `$defaults`, shouldn't it?

Reply:
Should be identical with the string in infrastructure-playbooks now